docker/run

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash -e
+
+groupadd -g ${GROUP_ID} geomapfish
+useradd --shell /bin/bash --uid ${USER_ID} --gid ${GROUP_ID} ${USER_NAME}
+
+su $USER_NAME -c "umask ${UMASK};$*"
```

Comment:
effectively :-)
Have you a better way to do that?
do it in python with a temporary file?